numpy/random/_generator.pyx

```diff
@@ -269,7 +269,7 @@ cdef class Generator:
 
         See Also
         --------
-        ~.Generator.uniform : Drawn samples from the parameterized uniform distribution.
+        `~.Generator.uniform` : Drawn samples from the parameterized uniform distribution.
```

Comment:
numpydoc does some pre-parsing on the See Also entries and can't currently handle things that start with `~` I believe.

I think it'll work correctly if you provide an explicit role, e.g.

```suggestion
        :meth:`~numpy.random.Generator.uniform` : Drawn samples from the parameterized uniform distribution.
```

That'll look good and link correctly in the html docs, but the source rst is kind of ugly so YMMV